src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/mapping/ClaimMapper.java

```diff
@@ -82,4 +84,12 @@ private Settlement toNullableSettlement(String input) {
         }
     }
 
+    private <T> T toNullableEntity(String input, TypeReference<T> entityClass) {
```

Comment:
I can see this method duplicated in two places, is there an option to reuse after moving in common place?

Reply:
Its duplicated in the migration project which currently has everything duplicated, out of scope for this PR

Reply:
agree, we can leave it as it is